changelog/1.36.0.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "1.36.0"
+description: "Released on 10/26/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.36.0
+
+### Features ✨
+
+- infra: Added sessionAffinity to coderd service.
+- infra: Allowed older verions of curl for self-contained builds.
+- infra: Allowed workspace template builds even if user does not have permission
+  to add webhooks on their git repo.
+- infra: Broke out API request latency Prometheus metrics by route.
+- infra: Added Prometheus metrics around background job execution.
+- infra: Added support for cached CVMs with 5.15 and 5.16 kernel versions.
+
+### Bug fixes 🐛
+
+- web: Fixed the "back to site" button that appears when trying to access an IDE
+  on a workspace that has shut off.
+- infra: Fixed a goroutine leak.
+- web: Fixed relative links on embedded docs.
+- infra: Large numbers of concurrent workspace builds are distributed more
+  evenly amongst multiple coderd replicas.
+- web: Allow org members to view activity metrics.
+- infra: Prevent a panic when updating a template.
+- infra: Return the updated user after an update instead of the cached user.
+- infra: Fix SSH connections not being audit logged.
+- infra: Fixed an issue where temporary pods created during build did not have
+  templates applied.
+- infra: Fixed an issue where the hostname was set to `workspace` for CVM
+  workspaces irrespective of the workspace name.
+
+### Security updates 🔐
+
+- infra: Fixed an issued where Coder services inside the workspace could be
+  reached via the network from outside in some environments.
+
+### Known Issues ℹ️
+
+- On AWS EKS, cached CVMs are known to not work with more recent versions of the
+  `Ubuntu2004` AMI family that include kernel version `5.15-aws`. Workaround:
+  roll back to a version that includes an earlier kernel version (for example,
+  `ubuntu-eks/k8s_1.22/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20220706`)
+  or build a custom AMI with the generic kernel.
```

Comment:
☝️

Reply:
Out of curiosity, does 22.04 work fine?

Reply:
> Out of curiosity, does 22.04 work fine?

I'm not aware of any 22.04 AMIs published by Amazon.